charts/portal/README.md

```diff
@@ -102,7 +102,7 @@ This section describes configurable parameters in **values.yaml**, there is also
 |-------------------------------------------|----------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
 | `portal.domain` | The Portal Domain | `example.com` |
 | `portal.enrollNotificationEmail` | Notification email address | `noreply@example.com` |
-| `portal.analytics.enabled` | Enable/Disable the Druid Analytics stack | `true` |
+| `portal.analytics.enabled` | Enable/Disable the Druid Analytics stack. Ensure to apply the minio-secret manually if the flag is set to true during 'helm upgrade' | `true` |
```

Comment:
provide a sample yaml file in utils. Also, document the command and instructions to create the secret using that file.

Reply:
Done